src/modelfree/score_agent.py

```diff
@@ -53,23 +53,28 @@ def get_empirical_score(_run, env, agents, episodes, render=False):
 @score_ex.config
 def default_score_config():
     env_name = 'multicomp/SumoAnts-v0'  # Gym env ID
-    agent_a_type = 'zoo'   # type supported by policy_loader.py
-    agent_a_path = '1'     # path or other unique identifier
-    agent_b_type = 'zoo'   # type supported by policy_loader.py
-    agent_b_path = '2'     # path or other unique identifier
-    num_env = 1            # number of environments to run in parallel
-    episodes = 20           # number of episodes to evaluate
-    render = True          # display on screen (warning: slow)
-    videos = False         # generate videos
-    video_dir = 'videos/'  # video directory
+    agent_a_type = 'zoo'                # type supported by policy_loader.py
+    agent_a_path = '1'                  # path or other unique identifier
+    agent_b_type = 'zoo'                # type supported by policy_loader.py
+    agent_b_path = '2'                  # path or other unique identifier
+    record_traj = False                 # whether to record trajectories
```

Comment:
Apropos of the fact that we currently have our default config not using the trajectory recording functionality, do we have or, if we don't have, do we want a test that start up a TrajectoryRecording env, confirm that a JSON is saved properly, that is the expected set of keys, etc? My default would be to think that's a good thing to add, unless there's some general belief to not test things that require running trajectories because that'd be too costly time wise

Reply:
[Ignore this, found tests after scrolling down more]